ibm_catalog.json

```diff
@@ -0,0 +1,186 @@
+              {
+                "service_name": "iam-access-groups",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::role:Editor"
+                ]
+              },
+              {
+                "service_name": "iam-identity",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::role:Operator"
+                ]
+              },
+              {
+                "service_name": "resource-group",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::role:Viewer"
+                ]
+              },
+              {
+                "service_name": "secrets-manager",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Administrator",
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ]
+              },
+              {
+                "service_name": "cloud-object-storage",
+                "notes": "[Optional] Required if you are enabling VPC flow logs",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                  "crn:v1:bluemix:public:iam::::role:Editor"
+                ]
+              }
+            ],
+            "configuration": [
+              {
+                "key": "ibmcloud_api_key"
+              },
+              {
+                "key": "existing_secrets_manager_crn",
+                "required": true
+              },
+              {
+                "key": "prefix",
+                "required": true
+              },
+              {
+                "key": "ibmcloud_cis_api_key"
+              },
+              {
+                "key": "internet_services_crn"
+              },
+              {
+                "key": "internet_services_account_id"
+              },
+              {
+                "key": "internet_service_domain_id"
+              },
+              {
+                "key": "dns_config_name"
+              },
+              {
+                "key": "ca_config_name"
+              },
+              {
+                "key": "lets_encrypt_environment"
+              },
+              {
+                "key": "acme_letsencrypt_private_key"
+              },
+              {
+                "key":"private_key_secrets_manager_secret_crn"
+              },
+              {
+                "key":"skip_iam_authorization_policy",
+                "options": [
+                  {
+                    "displayname": "true",
+                    "value": "true"
+                  },
+                  {
+                    "displayname": "false",
+                    "value": "false"
+                  }
+                ]
+              }
+            ],
+            "dependencies": [
+              {
+                "name": "deploy-arch-ibm-secrets-manager",
+                "id": "5f149ae6-d781-460a-977d-c8c4171ad0fa-global",
+                "version": "^v1.3.0",
```

Comment:
this addons config is not up to date - please use the approach we have been rolling out to other DAs